src/main/java/uk/gov/ida/matchingserviceadapter/services/AttributeQueryService.java

```diff
@@ -45,31 +42,19 @@ public void validate(AttributeQuery attributeQuery) {
     }
 
     public void validateAssertions(String expectedInResponseTo, List<Assertion> assertions) {
-
-        if (hasMdsAssertion(assertions)) {
-            verifyAssertionService.validate(expectedInResponseTo, assertions);
-        } else {
-            eidasAssertionService.validate(expectedInResponseTo, assertions);
-        }
+        verifyAssertionService.validate(expectedInResponseTo, assertions);
     }
 
     private boolean hasMdsAssertion(List<Assertion> assertions) {
         return assertions.stream()
-                    .map(assertionClassifier::getClassification)
-                    .anyMatch(MDS_ASSERTION::equals);
+                .map(assertionClassifier::getClassification)
```

Comment:
I don't think `hasMdsAssertion` is used anymore - can probably drop it.

Reply:
Agree, removing